content/docs/reactivesearch/react-searchbox/searchcomponent.md

```diff
@@ -0,0 +1,339 @@
+---
+title: 'SearchComponent API Reference'
+meta_title: 'Documentation for SearchComponent'
+meta_description: 'React SearchBox is a lightweight react search library with some common utilities.'
+keywords:
+    - react-searchbox
+    - search library
+    - elasticsearch
+sidebar: 'docs'
+nestedSidebar: 'react-searchbox-reactivesearch'
+---
+
+## SearchComponent
+
+### Props
+
+#### To configure the Appbase environments
+
+-   **index** `string` [required]
```

Comment:
For all the props, refer to the issues highlighted in the SearchBox documentation review.